Makefile

```diff
@@ -250,6 +250,12 @@ flow/flow.pb.go: flow/flow.proto filters/filters.proto
 	sed -e 's/type Flow struct {/type Flow struct { XXX_state flowState `json:"-"`/' -i $@
 	# to fix generated layers import
```

Comment:
remove comment

Reply:
This comment was already there and it probably still makes sense (as the following line relates to layers import was already there too)

Reply:
k